src/Microsoft.Extensions.Primitives/StringValues.cs

```diff
@@ -253,6 +254,256 @@ public static StringValues Concat(StringValues values1, StringValues values2)
             return new StringValues(combined);
         }
 
+        public static bool Equals(StringValues objA, StringValues objB)
```

Comment:
The parameters should be named `left` and `right` per the framework design guidelines.